cmd/manager/main.go

```diff
@@ -69,6 +71,7 @@ func main() {
 	flag.IntVar(&pprofPort, "pprof-port", 6060, "The port that the pprof server binds to if enabled.")
 	flag.Float32Var(&rateLimitQps, "qps", 20.0, "The client-side token bucket rate limit qps.")
 	flag.IntVar(&rateLimitBurst, "burst", 30, "The client-side token bucket rate limit burst.")
+	flag.BoolVar(&multiClusterElection, "multi-cluster-election", false, "Enable multi-cluster leader election.")
```

Comment:
Standard gripe against boolean in apis. Would rather have an enum anticipating the likely direction of expansion. Eg multiClusterElectionType with values Disabled and Standard or V1.